dbms/src/test/java/org/polypheny/db/constraints/ForeignKeyConstraintTest.java

```diff
@@ -401,16 +391,18 @@ public void testUpdateOutConflict() throws SQLException {
     }
 
 
-    @Test
-    public void testUpdateInNoConflict() throws SQLException {
+    @ParameterizedTest(name = "{index}. Create Index: {0}")
+    @ValueSource(booleans = { false, true })
+    @Tag("cottontailExcluded") // only with indexes
```

Comment:
Why is this cottontail exclude necessary?

Reply:
There seems to be some weird behavior with cottontail, when indexes are enabled for a table and you do some referential updates, where it cannot find the columns anymore. As we still on an older version of cottontail, I would not put additional effort into it and fix it when we update it in a later PR.